architecture/additional_concepts/authorization.adoc

```diff
@@ -880,6 +880,43 @@ in their SCC set.  This allows cluster administrators to run pods as any
 user by without specifying a `RunAsUser` on the pod's `SecurityContext`.  The
 administrator may still specify a `RunAsUser` if they wish.
 
+[[rbac-to-scc]]
+==== Role-Based Access to SCCs ====
+Starting with {product-title} 3.11, you can specify Security Context Constrains as a
+resource that is handled by RBAC. This allows you to
+scope the access to your SCCs to a certain project or cluster wide. Assigning users/groups
+directly to an SCC retains cluster-wide scope.
+
+To include access to SCCs for your role, you need to specify the following rule:
+.Role-Based Access to SCCs
+[source,yaml]
+----
+rules:
+  apiGroups:
+  - security.openshift.io <1>
+  resources:
+  - securitycontextconstraints <2>
+  verbs:
+  - use
+  resourceNames:
+  - myPermittingSCC <3>
+----
+<1> The API group which includes the `securitycontextconstraints` resource
```

Comment:
s/which/that

Reply:
Fixed. Just out of curiosity as a non-native English speaker - what's the difference here?

Reply:
Thanks!

You use "that" if the information is necessary to the meaning of the sentence and "which" if the info is optional, clarifying information. You set clauses that begin with "which" off with commas. So, "Let's go to Starbucks, which has fake macchiattos, to get a coffee." versus "Let's go to the Starbucks that is across the street." If we were going to meet for coffee, my opinion on their coffees wouldn't matter, but the location would.

In conversation or informal writing, you can really use either. Most people will understand exactly what you mean. But this is a formal context, so we have to try to introduce clauses correctly.

Reply:
Ah, I see! We were always taught that you can really use either when talking about things. I wasn't sure why it's suddenly different, given your explanation, so I went digging through the Internets and finally https://blog.oxforddictionaries.com/2012/09/07/that-who-which/ explains that this is a difference between American and British English. But nobody ever told us of it :slightly_smiling_face: 

Oh, those things you learn every day! I'll be meeting one of my high school English teachers real soon, this is a fact that I have to share with her. Thank you for your explanation!

Reply:
That makes sense! I didn't realize that there was a difference in usage for American and British English here. The rules of English are baffling enough without introducing these odd differences, and I am grateful that the internet could finish my explanation. :slightly_smiling_face: 

I'm glad I could provide your odd fact for the day and that you keep up with your teachers! I'm curious what she says about this issue. I am always happy to help!